zeppelin-web/src/app/notebook/paragraph/paragraph.controller.js

```diff
@@ -796,7 +796,7 @@ function ParagraphCtrl ($scope, $rootScope, $route, $window, $routeParams, $loca
                   name: v.name,
                   value: v.value,
                   meta: v.meta,
-                  caption: computeCaption(v.value, v.meta),
+                  caption: computeCaption(v.name, v.meta),
```

Comment:
Is this a bug fix ?

Reply:
before that name and value have always were always the same so this is more of an improvement. To be able to insert the value of the additional characters for user convenience.